src/widgets/index/index.ts

```diff
@@ -174,7 +168,11 @@ const index = (props: IndexProps): Index => {
       const initialSearchParameters =
         // Uses the `searchParameters` for the top level index only, it allows
         // us to have the exact same behaviour than before for the mono-index.
-        parent === null ? instantSearchInstance._searchParameters : {};
+        parent === null
+          ? new algoliasearchHelper.SearchParameters(
+              instantSearchInstance._searchParameters
```

Comment:
We could have nest the condition inside the argument, that would ease the reading.